LiteCore/Database/LegacyAttachments.cc

```diff
@@ -92,6 +98,14 @@ namespace litecore { namespace legacy_attachments {
         Encoder enc;
         enc.setSharedKeys(sk);
         enc.writeValue(root, [&](const Value *key, const Value *value) {
+            {
+                // CBL-4944
+                if (const Dict* valDict = Value::asDict(value); C4Blob::isBlob(FLDict(valDict))) {
+                    if (!valDict->get("length"_sl)) {
+                        C4Warn("Find blob with nil length in legacy_attachments::encodeStrippingOldMetaProperties: %s", valDict->toJSONString().c_str());
```

Comment:
CBSE case has attachment length = 0 while it shouldn't be 0. The attachment length = 0 could be valid for empty blob but for the purpose of the investigation, we should log as well.